assets/js/admin/course-edit.js

```diff
@@ -44,32 +44,36 @@ domReady( () => {
 
 	jQuery( '#course-prerequisite-options' ).select2( { width: '100%' } );
 
-	function trackLinkClickCallback( event_name ) {
-		return function () {
-			var properties = {
-				course_status: jQuery( this ).data( 'course-status' ),
-			};
+	const trackLinkClickCallback = ( event_name ) => ( e ) => {
+		var properties = {
+			course_status: e.target.dataset.courseStatus,
+		};
 
-			// Get course status from post state if it's available.
-			if ( wp.data && wp.data.select( 'core/editor' ) ) {
-				properties.course_status = wp.data
-					.select( 'core/editor' )
-					.getCurrentPostAttribute( 'status' );
-			}
+		// Get course status from post state if it's available.
+		if ( wp.data && wp.data.select( 'core/editor' ) ) {
+			properties.course_status = wp.data
+				.select( 'core/editor' )
+				.getCurrentPostAttribute( 'status' );
+		}
 
-			sensei_log_event( event_name, properties );
-		};
-	}
+		sensei_log_event( event_name, properties );
+	};
 
 	// Log when the "Add Lesson" link is clicked.
-	jQuery( 'a.add-course-lesson' ).click(
-		trackLinkClickCallback( 'course_add_lesson_click' )
-	);
+	document
+		.querySelector( 'a.add-course-lesson' )
+		.addEventListener(
+			'click',
+			trackLinkClickCallback( 'course_add_lesson_click' )
+		);
```

Comment:
A bit late to the party, but I've noticed a `Cannot read properties of null (reading 'addEventListener')` error generated by this line when creating a new course.

Reply:
Good catch! Fixed here: https://github.com/Automattic/sensei/pull/4858